cms/templates/cms/program_page.html

```diff
@@ -63,17 +63,23 @@ <h1 class="title">
                   href="{{ page.program_subscribe_link }}"
                   target="blank">Learn More</a>
         {% else %}
-          {% if not authenticated %}
-            <a href="{% url 'signin' %}?program={{page.program.id}}" class="mdl-button hero-button mdl-cell--hide-phone">
-              Create Account
-            </a>
-            <div class="log-in mdl-cell--hide-phone">
-              Already a Member?
-              <a href="{% url 'signin' %}?program={{page.program.id}}">
-                Sign In
-              </a>
-            </div>
-          {% endif %}
+            {% if page.program.id == 2 %}
+                <a href="https://mitxonline.mit.edu/" class="mdl-button hero-button mdl-cell--hide-phone">
```

Comment:
What do you think is the best URL to use for this? Could we go directly to the create-account page?